www/install/php/Update-20.10.0-beta.2.post.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+include_once __DIR__ . "/../../class/centreonLog.class.php";
+$centreonLog = new CentreonLog();
+
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 20.10.0-beta.2.post : ';
+
+/**
+ * Queries needing exception management and rollback if failing
+ */
+try {
+    $pearDB->beginTransaction();
+    // Remove data inserted in 20.10.0-beta1
+    $pearDB->query("DELETE FROM `platform_topology`");
+
+    /**
+     * register server to 'platform_status' table
+     */
+    // Check if the server is a Remote or a Central
+    $type = 'central';
+    $serverType = $pearDB->query("
+        SELECT `value` FROM `informations`
+        WHERE `key` = 'isRemote'
+    ");
+    if ('yes' === $serverType->fetch()['value']) {
+        $type = 'remote';
+    }
+    // Check if the server is enabled
+    $errorMessage = "Unable to find the server in 'nagios_server' table.";
+    $serverQuery = $pearDB->query("
+        SELECT `id`, `name` FROM nagios_server
+        WHERE localhost = '1' AND ns_activate = '1'
+    ");
+
+    $bindValues = [];
+    $hostName = gethostname();
+    $hostName
+        ? $bindValues[':hostname'] = [\PDO::PARAM_STR => $hostName]
+        : $bindValues[':hostname'] = [\PDO::PARAM_NULL => null];
```

Comment:
```suggestion
    $hostName = gethostname() ?: null;
    $bindValues[':hostname'] = [\PDO::PARAM_STR => $hostName];
```

Reply:
@jeremyjaouen ... it seems it's the right time to remove the foreach, foreach loop ;)